restore-sql-server-transaction-logs/Function/env.yml

```diff
@@ -0,0 +1,7 @@
+INSTANCE_ID: ""
```

Comment:
(this comment is valid for the JSON file below, and it's an opinion)

I'm not fond of including "template configuration files" like this one and the JSON below in source repositories. This leads to some duplication (what happens when you change the key of a value in code, but you forget to update these example template files? Is there linting in place to guard against this?)

Also, these files are mixing default values and values that users are supposed to change.

I think that these scripts should:

- For values that you have default values that the user can eventually override, define these defaults in code, and read the config file to override what the user specified.
- For values that the user needs to provide (the ones in the JSON file?): fail if they don't provide these values, with explicative error messages.

Reply:
SGTM